sh/tests/file-checker_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+submitted=$(bash "$script_dirS"/student/plus.sh $1)
```

Comment:
Why is this testing "plus.sh"?
also even if you change it to the correct exercise name it doesn't work
Are you testing it locally?

Reply:
Hey, my bad, that was a copy-paste mishap, I might have undone the changes without realising, or replaced my changes with that, I don't know the reason, but that's not what I wanted to be reviewed. I fixed it already.

Reply:
It happens no worries